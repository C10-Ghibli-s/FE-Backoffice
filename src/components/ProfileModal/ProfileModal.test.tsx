import { ProfileModal } from ".";
import renderer from "react-test-renderer";

it("should render modal elements correctly", () => {
  const tree = renderer
    .create(
      <ProfileModal openModal={false} setOpenModal={false}/>
    )
    .toJSON();
  console.log(tree);
  expect(tree).toMatchSnapshot();
});
