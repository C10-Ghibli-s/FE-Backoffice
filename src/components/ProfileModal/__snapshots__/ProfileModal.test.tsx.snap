// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render modal elements correctly 1`] = `null`;
